### PR TITLE
taxonomy: Added some Dutch synonyms and translations

### PR DESCRIPTION
**Pull request by Annieieieie**
### What

Added some Dutch synonyms and translations

### Related issue(s) and discussion

- Fixes #[ISSUE NUMBER]

### CODE REVIEW

File: taxonomies/old/containers.txt [hunk 1]
```diff
@@ -63,18 +64,21 @@ wikidata:en:Q2846150
...
 de:Kanne
+nl:Potje
 pt:Pote, boião
 
 en:Tray
 bg:Тарелка
 de:Körbchen
 fr:Barquette
 hu:Tálca
+nlBakje, schaaltje
```
Comment (danizavtz) [inline taxonomies/old/containers.txt:81]: I think you are missing a two-dots (":") here.

File: taxonomies/old/containers.txt [hunk 2]
```diff
@@ -107,45 +112,52 @@ bg:Корк
...
-nl:Dop
+nl:Dop, kurk
 pt:Cortiça
 hu:Dugó
 wikidata:en:Q49444
 
 fr:Puisette
 hu:Fa
 
+
```
Comment (danizavtz) [inline taxonomies/old/containers.txt:123]: I my opinion this blank line can be removed, it is generating a diff in the code difficulting review for a thing that does not changed.

**Review by danizavtz — COMMENTED** [review]
Just added some comments.

**Comment by aleene** [comment]
Noticed that you made changes to the folders `old` and `unused`. Probably not useful at the moment. Not sure why these exist.

**Comment by stephanegigandet** [comment]
The packaging_shapes taxonomy test fails because "pak" is associated to both en:pack and en:box.

ERROR - nl:pak already is a synonym of nl:pak for entry en:pack - nl:pak cannot be mapped to entry en:box / nl:doos

Unfortunately we cannot have the same translation for 2 different entries.

**Comment by aleene** [comment]
Remove nl:Pak then.

File: taxonomies/old/containers.txt (hunk 2 above)
Comment (alexgarel) [reply taxonomies/old/containers.txt:123]: @danizavtz FYI, in github when you review PRs you can set not to see white space changes
Capture d’écran du 2022-09-06 16-18-29

Comment (danizavtz) [reply taxonomies/old/containers.txt:123]: Thanks, I'll do it next review.

**Review by stephanegigandet — APPROVED** [review]
Thank you!

**Pull request merged**